boards/microchip/sam/sama7g54_ek/pre_dt_board.cmake

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2025 Microchip Technology Inc. and its subsidiaries
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Suppress "unique_unit_address_if_enabled" to handle the following overlaps:
+# - /soc/ethernet@e2800000 & /soc/mdio@e2800000
+# - /soc/ethernet@e2804000 & /soc/mdio@e2804000
```

Comment:
if they share registers, its common to have a common parent node, like the stm32 ethernet has